backend/src/main/java/com/ody/eta/service/EtaService.java

```diff
@@ -40,51 +41,38 @@ public Eta saveFirstEtaOfMate(Mate mate, RouteTime routeTime) {
     public MateEtaResponses findAllMateEtas(MateEtaRequest mateEtaRequest, Long meetingId, Member member) {
         Mate requestMate = findByMeetingIdAndMemberId(meetingId, member.getId());
         Meeting meeting = requestMate.getMeeting();
-        LocalDateTime meetingTime = TimeUtil.trimSecondsAndNanos(meeting.getMeetingTime());
         Eta mateEta = findByMateId(requestMate.getId());
-        LocalDateTime now = TimeUtil.nowWithTrim();
+        mateEta.updateMissingBy(mateEtaRequest.isMissing());
 
-        if (mateEtaRequest.isMissing()) {
-            mateEta.updateRemainingMinutes(-1L);
-        }
-
-        if (determineArrived(mateEtaRequest, meeting, now)) {
+        if (determineArrived(mateEtaRequest, meeting) && !mateEta.isMissing()) {
             mateEta.updateArrived();
         }
-
-        if ((!mateEta.isArrived() && isOdysayCallTime(mateEta)) &&
-                !(mateEtaRequest.currentLatitude() == null || mateEtaRequest.currentLongitude() == null)) {
+        if ((!mateEta.isArrived() && isRouteClientCallTime(mateEta)) && !mateEta.isMissing()) {
```

Comment:
[제안]

모두 `&&` 조건이라 괄호는 불필요할 것 같아요.
`isMissing()`을 앞쪽에 두는 게, 미세하지만(?) 성능이 더 좋을 것 같아요. false이면 `isRouteClientCallTime`를 연산하지 않을 테니까요.
```suggestion
        if (!mateEta.isArrived() && !mateEta.isMissing() && isRouteClientCallTime(mateEta)) {
```

Reply:
더 좋은 카디널리티네요! 반영해보겠습니다!

Reply:
이 전 조건문에 사용됐던 괄호가 들어가버렸네요 😅
괄호 제거 후 순서도 변경했습니다 ~👍🏻